README.md

```diff
@@ -34,20 +34,15 @@ Manim is an animation engine for explanatory math videos. It's used to create pr
 -  [License](#license)
 
 ## Installation
-
+> **WARNING:** These instructions are for the community version _only_. Trying to use these instructions to install [3b1b/manim](https://github.com/3b1b/manim) or instructions there to install this version will cause problems. Read [this](https://docs.manim.community/en/stable/installation/versions.html) and decide which version you wish to install, then only follow the instructions for your desired version.
+        
```

Comment:
- Given that the differences between manim versions are already listed under the docs, I don't think this particular section is necessary (maybe it should be made clearer there?). 

- Also, imo most people that are viewing the manimce repository will be trying to run code from it. If they're already here (most likely linked from 3b1b/manim) then this feels redundant.

```suggestion
```

Reply:
I think this is important enough to warrant reiterating, but I see your point. Let's see what others think.

Reply:
The trouble I see here is that the 3b1b/manim README.md isn't giving the same warning so without cooperation, we'd still likely face people using 3b1b/manim and running examples from the ManimCommunity documentation if somehow they ended up here.

I think it's enough to have this solely in our documentation and make it more clear there if we're still facing issues with people using incompatible code with the different versions of Manim. Somehow people are still using v0.1.1 documentation and that's more of an issue, but we already have a note/warning for viewing older versions of the docs.

Reply:
I think it is a good idea to add the information in the README as well; there is little harm with replicating this information.